src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -15,23 +15,17 @@ import {
 import { TutorialSchema, tutorialSchema } from './lib/tutorial_schema';
 import { builtInTutorials } from '../../tutorials/register';
 import { CustomIntegrationsPluginSetup } from '../../../../custom_integrations/server';
-import { Category, CATEGORY_DISPLAY } from '../../../../custom_integrations/common';
+import { Category } from '../../../../custom_integrations/common';
```

Comment:
nit: I think renaming this to `IntegrationCategory` would be helpful for these usages that are outside the custom_integrations plugin so it's clear what this is for.